rules/BUILD.bazel

```diff
@@ -41,3 +41,35 @@ config_setting(
         ":local_debug_options_enabled": "True",
     },
 )
+
+config_setting(
```

Comment:
there are already config settings in rules/apple_platform/BUILD.bazel you should be able to use

Reply:
✅